<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventilate_newuser</class>
 <widget class="QDialog" name="ventilate_newuser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>212</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Ventilate: New User</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(66, 66, 66);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="minimumSize">
      <size>
       <width>461</width>
       <height>84</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>84</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>461</width>
       <height>84</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/WelcometoVentilate.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="usernameLabel">
       <property name="minimumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Username: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="usernameText">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="minimumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Password: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordText">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="minimumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Email: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="emailText">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(66, 66, 66);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ventilate_newuser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ventilate_newuser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
